src/main/resources/application-prod.properties

```diff
@@ -5,7 +5,7 @@ spring.jpa.properties.hibernate.format_sql=true
 
 spring.data.couchbase.auto-index=true
 # address of the Couchbase Cluster nodes
-spring.couchbase.bootstrap-hosts=couch-dakar
+spring.couchbase.bootstrap-hosts=dakar_couch
```

Comment:
I can't find it on docker compose file

Reply:
on the compose file, you'll find the "couch" service. But the final hostname will be a combination of the swarm stack and the service name, means: dakar_couch